src/converters/notion/index.ts

```diff
@@ -0,0 +1,232 @@
+import { parse } from 'csv-parse/sync';
+import { TanaIntermediateAttribute, TanaIntermediateFile, TanaIntermediateNode } from '../../types/types';
+import { idgenerator } from '../../utils/utils';
+import { IConverter } from '../IConverter';
+
+const isPageLink = (s: string) => {
+    return !!s.match(/\.md$/)
+}
+
+const parseFileContents = (fileContent: string): Record<string, string>[] => {
+    // Looks like notion exporter inserts a Zero Width No-Break Space character at the start of the file.
+  return parse(fileContent.slice(1), {columns: true, skip_empty_lines: true, encoding: 'utf8', })
+}
+
+const createChildrenForRoot = (parsedContent: Record<string, string>): TanaIntermediateNode[] | undefined => {
+    const keys = Object.keys(parsedContent)
+    const fields = keys.map(key => {
+        const newField = {
+            uid: idgenerator(),
+            name: key,
+            createdAt: new Date().getTime(),
+            editedAt: new Date().getTime(),
+            type: 'field',
+            todoState: undefined, 
+        }
+
+        const fieldChildren = [{
+            uid: idgenerator(),
+            name: parsedContent[key],
+            createdAt: new Date().getTime(),
+            editedAt: new Date().getTime(),
+            type: 'node',
+            todoState: undefined,
+        }]
+
+        return {...newField, children: fieldChildren}
+    }) as TanaIntermediateNode[]
+
+    return fields
+}
+
+const createRootNode = (parsedContent: Record<string, string>): TanaIntermediateNode => {
+    const newNode = {
+        uid: idgenerator(),
+        name: parsedContent['Name'],
+        createdAt: new Date().getTime(),
+        editedAt: new Date().getTime(),
+        type: 'node',
+        todoState: undefined,
+    } as const
+
+    const children = createChildrenForRoot(parsedContent)
+    return {...newNode, children}
+}
```

Comment:
this creates a root node for each of the objects.

We statically use the `Name` column to determine the `node.name` of the root node.

Afaict, every notion table has a `Name` column. If not, this will need to be a param passed in from the CLI.